pkg/build/make.go

```diff
@@ -64,7 +64,7 @@ func K8sDir(topdir string, parts ...string) (string, error) {
 // TODO(RonWeber): This whole untarring and cd-ing logic is out of
 // scope for Build().  It needs a better home.
 func extractBuiltTars() error {
-	allBuilds, err := K8sDir("kubernetes", "_output", "gcs-stage")
+	allBuilds, err := K8sDir("kubernetes", "bazel-bin", "build", "release-tars")
```

Comment:
The build logic in general needs some refactoring, but let's fix it the other way around.
since we already have a separate bazel builder let's modify this one to have 
`target = "quick-release"`

Reply:
Fixed as suggeset. :+1: